ethcontract-common/Cargo.toml

```diff
@@ -12,7 +12,7 @@ Common types for ethcontract-rs runtime and proc macro.
 """
 
 [dependencies]
-ethabi = "13.0"
+ethabi = { git = "https://github.com/vkgnosis/ethabi.git", rev ="f23954bb5b86687f0d780935b2eacc503393dc44" }
```

Comment:
Not sure what the best way to distribute this is. Could keep on git, push to crates, include the whole crate in this repo (maybe as a submodule).

Reply:
> Could keep on git

The issue there is that we can't publish to Crates.io then.